files/templates/scrapeconfigs/additional-scrape-configs.template.yaml

```diff
@@ -655,6 +655,10 @@
   - source_labels: [__name__]
     regex: (nginx_ingress_controller_bytes_sent_bucket|nginx_ingress_controller_request_duration_seconds_bucket|nginx_ingress_controller_request_size_bucket|nginx_ingress_controller_response_duration_seconds_bucket|nginx_ingress_controller_response_size_bucket)
     action: drop
+  # on workload clusters only keep metrics from namespaces we manage
+  - source_labels: [namespace]
+    regex: (kube-system|giantswarm.*)
+    action: keep
```

Comment:
Wait this apply to every cluster, could we re-use the condition we saw ... somewhere ... ?

Reply:
It's in the workload job, so I assumed it's only WCs? Or do you mean you'd want to limit it further?

Reply:
But I think you're right, I don't actually see anything that would limit this to WCs only.

Reply:
That's a MC job too. I'm not sure we should limit this on MCs.

Reply:
Yeah, you're right. OK leave it with me, I'll play around with the conditionals and test this a bit more. Thanks for checking, and well spotted!

Reply:
Confirmed on _gorgoth_. Before this change:

![prometheus-mc workload-job-apps-from-other-namespaces before-change 2021-07-13--16-49-29](https://user-images.githubusercontent.com/42101611/125486616-8373247e-ee6b-45da-a908-90b4bbc19de9.png)

After:

![prometheus-mc workload-job-apps-from-other-namespaces after-change-1 2021-07-13--16-57-52](https://user-images.githubusercontent.com/42101611/125486663-26054ed7-249f-40bd-b2ff-65870bf1ef21.png)

Reply:
OK with commit 9470f0700a636a85abe14e8706ed3d00f6baa7b4 we're back to:

![prometheus-mc workload-job-apps-from-other-namespaces after-change-2 2021-07-14--09-49-42](https://user-images.githubusercontent.com/42101611/125593487-21360616-9c13-4b4c-9873-45c7cd12c96b.png)